pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -57,7 +57,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook makeWrapper pkg-config doxygen ];
   buildInputs = [ augeas dnsutils c-ares curl cyrus_sasl ding-libs libnl libunistring nss
                   samba nfs-utils p11-kit python3 popt
-                  talloc tdb tevent ldb pam openldap pcre2 libkrb5
+                  talloc tdb tevent ldb pam openldap pcre2 libkrb5 libverto
```

Comment:
or that

Reply:
libverto does appear to be necessary for krb5...

Reply:
In the end I was able to get rid of it.